### DISCUSSION TITLE
additional sound module

### DISCUSSION THREAD

**Discussion by psicho100pro** [OpenI6X/opentx#261] [Ideas]
Hi,
on the thread on rcgroups I noticed that at the beginning of the project the sound processing for this TX was solved.
A few days ago this was offered, maybe it has some value. 🤷‍♂️
www.youtube.com/watch?v=m-MPBjScNRk

**Comment by ajjjjjjjj** [comment]
Hi, there is no problem to output audio, but flash space needed to actually enable voice and sdcard support.

**Comment by psicho100pro** [comment]
Yes I know, this module is dealing with SD card storage. The link below the video also contains sketch that has a few lines. The individual sounds are called // audio.connecttoFS (SD, "/ MYMUSIC.mp3"); Something like a remote control mp3. ?
In native openTX, the call is probably handled by Enumeration. ? I'll find it in the code in a few years. :D

**Reply by ajjjjjjjj** [reply]
Yes, there is a way to use external module like in Erfly6, but i'm not interested into implementing that, sorry. This radio have some limits and better way would be to get a officially supported EdgeTX device.

**Reply by psicho100pro** [reply]
Don't take it that I'm trying to convince you to solve this thing. Not really. I personally don't use sound at all. 
I guess I have such a deviation that I always try to solve things every way.
Maybe I'll publish flight mode buttons (radiomaster) here, but it's no longer a matter of firmware and it's similar to the solution I already designed the multiswitch. If anyone wants.

But still, you might consider placing a donation button in the project description.
Why?
I see a response to your work on youtube and your work may soon appear stolen somewhere on an aliexpress under the heading Wacharapol Hemachayart Ph.D (I feel like it's a professional github robber)
Good luck and greetings :)

**Comment by psicho100pro** [comment]
Already followed by Wacharapol a few days ago. 🤣
